client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/operations/OperationsFactory.java

```diff
@@ -56,6 +58,8 @@ public class OperationsFactory implements HotRodConstants {
 
    private Codec codec;
 
+   private Codec safePingVersionCodec;
```

Comment:
if the code is overwritten by `UntracedHotRodOperation`, why do we need this field?